packages/keystone/lib/List/index.js

```diff
@@ -1182,25 +1182,53 @@ module.exports = class List {
   async _validateHook(args, fields, operation, hookName) {
     const { originalInput } = args;
     const fieldValidationErrors = [];
+
     // FIXME: Can we do this in a way where we simply return validation errors instead?
-    args.addFieldValidationError = (msg, _data = {}, internalData = {}) =>
+    args.addValidationError = (msg, _data = {}, internalData = {}) => {
       fieldValidationErrors.push({ msg, data: _data, internalData });
-    await this._mapToFields(fields, field => field[hookName](args));
-    await this._mapToFields(
-      fields.filter(field => field.hooks[hookName]),
-      field => field.hooks[hookName](args)
-    );
+    };
+
+    // Compatibility. Remove at some point
+    args.addFieldValidationError = (...passthrough) => {
+      console.warn('addFieldValidationError is deprecated. Use addValidationError instead');
```

Comment:
👍

Reply:
please update the documentation as well ~

Reply:
I already replaced all mentions of `addFieldValidationError` with `addValidationError`.